README.md

```diff
@@ -131,14 +121,6 @@ Use your locally installed version of TypeScript.
 ```sh
 dtslint --localTs node_modules/typescript/lib types
 ```
-- `--expectOnly`

```

Comment:
expectOnly is used by dtslint-runner to speed up linting in incremental mode for dependents, so that only packages a PR modifies actually run all the lint rules.

Otherwise CI runs for node, react, et al take too long.